src/Components/Bookpage/index.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import BookDetails from "./BookDetails"
+
+function Bookpage() {
+    const { id } = useParams()
+    const [image, setImage] = useState("")
+    const [title, setTitle] = useState("")
+    const [author, setAuthor] = useState("")
+    const [year, setYear] = useState("")
+    const [pageCount, setPageCount] = useState("")
+    const [genre, setGenre] = useState("")
+    const [blurb, setBlurb] = useState("")
+    const [error, setError] = useState(false)
+
+    useEffect(() => {
+        fetch("https://book-swap-api.dev.io-academy.uk/api/books/" + id)
+            .then((res) => res.json())
+            .then((bookData) => {
+                if (bookData.message !== "Book successfully found") {
+                    setError(!error)
```

Comment:
Just use true here - this isn't intended to toggle the error, it's just switching it on. Using true makes that clearer

Reply:
done